### ISSUE TITLE
Wrong Summary & Return Value Descriptions for BigInteger.MinMagnitude & BigInteger.MaxMagnitude

### ISSUE THREAD

**Issue by zacharylayne** [dotnet/dotnet-api-docs#9525]
[BigInteger.MinMagnitude](https://learn.microsoft.com/en-us/dotnet/api/system.numerics.biginteger.minmagnitude?view=net-8.0) reads:
  **Summary:**  *Compares two values to compute which is lesser.*

  **x** 
    *The value to compare with y.*
   **y** 
    *The value to compare with x.*

  
  **Returns:** *x if it is less than y; otherwise, y.*

</br>

This would indicate the method is no different than [Min](BigInteger.Min).

</br>

Instead, **MinMagnitude** should be:

  **Summary:** *Returns the [BigInteger](https://learn.microsoft.com/en-us/dotnet/api/system.numerics.biginteger?view=net-8.0) with the smallest absolute value.*

  **Returns:** *The  [BigInteger](https://learn.microsoft.com/en-us/dotnet/api/system.numerics.biginteger?view=net-8.0) with the smallest absolute value. If the absolute values are equal, returns `x` if `x` < `0` and `y` if `x` >= 0.*

</br>

[BigInteger.MaxMagnitude](https://learn.microsoft.com/en-us/dotnet/api/system.numerics.biginteger.maxmagnitude?view=net-8.0) documentation has the same issue.

**Comment by gewarren** [comment]
Related to #9468.

**Comment by zacharylayne** [comment]
@gewarren Can I make the change myself? I think if I do, they will be overwritten by some API change, however the updates to INumberBase are not reflected in the API reference. Is this related to the type being forwarded or anything?

**Comment by gewarren** [comment]
> @gewarren Can I make the change myself? I think if I do, they will be overwritten by some API change, however the updates to INumberBase are not reflected in the API reference. Is this related to the type being forwarded or anything?

@zacharylayne Yes, you can make the change yourself. Since it's under the \<Docs> element, it won't be overwritten. Also, can you clarify what you mean by "the updates to INumberBase aren't reflected in the API reference"? I see the changes from #9468 here: https://learn.microsoft.com/en-us/dotnet/api/system.numerics.inumberbase-1.minmagnitude?view=net-8.0.

**Comment by zacharylayne** [comment]
@gewarren Apologies for the late reply. The fixes to the interface documentation do not seem complete, given that for `INumberBase<T>.MaxMagnitude`, `INumberBase<T>.MaxMagnitudeNumber` have the same return description. The same is true for their `Min` counterparts are the same, as well as `INumber<T>.Max`, `INumber<T>.MaxNumber`, etc.

The interface descriptions do not seem to have propagated to types implementing these interfaces within the `System` namespace. For example, `Int32.Max` and `Int32.MaxMagnitude`, which have the exact same summary, parameter, and return descriptions.

**Comment by gewarren** [comment]
@carlossanlop When you port documentation using your tool, what does it do with inheritdoc tags? Does it copy over the inheritdoc tag itself, or does it fill in the literal strings? If the latter, that might be a problem for cases like this where the inherited documentation has been updated, but the inheriting API already has literal strings, so it no longer pulls in the parent documentation even if it's different.

This seems to be where the original docs were copied over for this method: https://github.com/dotnet/dotnet-api-docs/commit/9f32d39cd411b39b7e7692a6e5de84adeb25b443

**Comment by gewarren** [comment]
> The fixes to the interface documentation do not seem complete, given that INumberBase<T>.MaxMagnitude and INumberBase<T>.MaxMagnitudeNumber have the same return value description. The same is true for their Min counterparts, as well as INumber<T>.Max & INumber<T>.MaxNumber, and INumber<T>.Min & INumber<T>.MinNumber.

@zacharylayne It would be great if you could update these, thanks!
